### DISCUSSION TITLE
Outside Layer with a Cloud Project

### DISCUSSION THREAD

**Discussion by Brosenieski** [opengisch/QField#2736] [Q&A]
Hello all,
i'm trying to use an outside layer as a basemap as described in the documentation here : https://docs.qfield.org/how-to/outside-layers/ and it's working fine if i export my project using a cable. However if i try to upload the same project in QFieldCloud i get this error:
![photo1649779685](https://user-images.githubusercontent.com/103019488/163007099-1f82e28f-52d0-4d1d-bb03-82ac5b46222d.jpeg)

Am i doing something wrong or the functionality to use an outside layer with a cloud project is yet to be implemented?
Thank you all for your help and the hardwork on Qfield.

**Comment by suricactus** [comment]
The outside layer functionality is yet to be implemented for QFieldCloud. If you want to feel interested to speed up the development of such feature, feel free to drop us a line on info at opengis.ch email.

**Comment by S1lk3** [comment]
@Brosenieski, did you find a workaround for this? I see this is still not implemented and I want to be able to push changes to the project via QFieldCloud without sending the basemap to the cloud (large file so too high price for one-time upload). See also https://github.com/opengisch/QField/discussions/4276. 

Thank you in advance.
